index.html

```diff
@@ -20,7 +20,6 @@ <h2>Software Engineer, <a href="//intelligencefusion.co.uk" target="_blank">Inte
 
     <ul>
       <li><a href="/about">About me</a></li>
-      <li><a href="/works">Recent projects</a></li>
```

Comment:
I might bring this back in the future, but for now, I would rather not have this page here because I haven't released anything new in a while.